osbuild-composer.spec

```diff
@@ -175,6 +176,7 @@ install -m 0755 -vp _bin/osbuild-weldr-tests                    %{buildroot}%{_l
 install -m 0755 -vp _bin/osbuild-dnf-json-tests                 %{buildroot}%{_libexecdir}/osbuild-composer-test/
 install -m 0755 -vp _bin/osbuild-image-tests                    %{buildroot}%{_libexecdir}/osbuild-composer-test/
 install -m 0755 -vp _bin/osbuild-auth-tests                     %{buildroot}%{_libexecdir}/osbuild-composer-test/
+install -m 0755 -vp _bin/osbuild-koji-tests                     %{buildroot}%{_libexecdir}/osbuild-composer-test/
```

Comment:
Should we introduce a new directory for test executables which are not base tests? I'm working on running all the files in `/usr/libexec/osbuild-composer-tests` as a "test" in `rpmci`. This is the code I use ATM (far from nice, but believe me, I want to make it nicer):
```
 sudo find /usr/libexec/osbuild-composer-tests/ -type f -exec sudo {} \\;"
```
where